spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  before(:each) do
+    @user = User.create(name: 'Tom', photo: '', posts_counter: 0)
+    @post = Post.create(
+      user: @user,
+      title: 'Here we are!',
+      text: 'Here they go',
+      comments_counter: 0,
+      likes_counter: 0
+    )
+    Comment.create(user: @user, post: @post, text: 'Hello bob')
+    Comment.create(user: @user, post: @post, text: 'Hello gram')
+    Comment.create(user: @user, post: @post, text: 'Hello sam')
+  end
+  describe 'GET /index' do
+    before(:example) { get user_posts_path(@user) } # get(:index)
+    it 'assigns all posts to @posts' do
+      get user_posts_path(@user)
+      expect(assigns(:posts)).to eq(Post.all)
+    end
```

Comment:
- [ ] This particular test fails as shown in the image below. Please try to fix it 👇 
![image](https://user-images.githubusercontent.com/46839548/191240977-2df4214a-91f7-4064-9c2b-a0fa02f5a689.png)

Reply:
No it doesn't fail for me